wgengine/filter/filtertype/filtertype.go

```diff
@@ -66,11 +66,23 @@ type CapMatch struct {
 // Match matches packets from any IP address in Srcs to any ip:port in
 // Dsts.
 type Match struct {
-	IPProto      views.Slice[ipproto.Proto] // required set (no default value at this layer)
-	Srcs         []netip.Prefix
+	// IPProto is the set of IP protocol numbers for which this match applies.
+	// It is required. There is no default value at this layer.
+	// If empty, it doesn't match.
+	IPProto views.Slice[ipproto.Proto]
+
+	// Srcs is the set of source IP prefixes for which this match applies.
+	Srcs []netip.Prefix
```

Comment:
say something about how it relates to SrcCaps and SrcContains.

Reply:
done